scripts/src/main/resources/scripts/command/tomcat

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+
+# autocompletion list
+if [ "${1}" = "shortlist" ]
+then
+  if [ -z "${2}" ]
+  then
+    echo "setup version help"
+  fi
+  exit
+fi
+
+
+
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+export CATALINA_HOME="${DEVON_IDE_HOME}/software/tomcat"
+TOMCAT_HOME="${DEVON_IDE_HOME}/software/tomcat"
+TOMCAT="${TOMCAT_HOME}/bin/startup.bat"
+TOMCAT="catalina.sh"
+if doIsWindows
+then
+  TOMCAT="catalina.bat"
+fi
+TOOL_VERSION_COMMAND="${TOMCAT_HOME}/bin/${TOMCAT} version"
+# shellcheck source=scripts/commandlet-cli
+source "$(dirname "${0}")"/../commandlet-cli
+
+# $1: optional silent mode
+function doSetup() {
+  doDevonCommand java setup silent
+  doInstall "tomcat" "${TOMCAT_VERSION}" "${1}"
+  return 0
+}
+
+function doStart() {
+   # dosetup silent
+   # doRunCommand "${TOMCAT_HOME}/bin/startup.bat"#
+   doTomcat start
+}
+
+function doStop() {
+    doTomcat stop
+}
+
+function doTomcat() {
+    doSetup silent
+    if doIsWindows
+    then
+      if [ "${1}" = "start" ]
+      then
+        "${TOMCAT_HOME}/bin/startup.bat"
+      elif [ "${1}" = "stop" ]
+      then
+        "${TOMCAT_HOME}/bin/shutdown.bat"
+      else
+        doFail "Unknown tomcat command: ${1}"
+      fi
+    elif doIsMacOs
+    then
+      "${TOMCAT_HOME}/bin/startup.sh" "${1}"
+    else
+      "${TOMCAT_HOME}/bin/startup.sh" "${1}"
+    fi
```

Comment:
Currently on Mac and Linux you always run startup no matter if ${1} is "start" or "stop" and `startup.sh stop` will still start tomcat and not stop it (I just tested this).
Your code for windows looks good.
Did you test the `*.sh` scripts on windows? If they also work, you do not need to distinguish the operating system at all what is a great simplification. According to my tests they work also on windows so we can ignore the *.bat files.
Otherwise go this way:
```
  doSetup silent
  local cmd
  if [ "${1}" = "start" ]
  then
    if doIsWindows
    then
      cmd="startup.bat"
    else
      cmd="startup.sh"
    fi
  elif [ "${1}" = "stop" ]
  then
    if doIsWindows
    then
      cmd="shutdown.bat"
    else
      cmd="shutdown.sh"
    fi
  else
    doFail "Unknown tomcat command: ${1}"
  fi
  "${TOMCAT_HOME}/bin/${cmd}"
```

Reply:
*.sh actually works on windows, so I shortened the code to: 
if [ "${1}" = "start" ]
then
     "${TOMCAT_HOME}/bin/startup.sh"
elif [ "${1}" = "stop" ]
then
  "${TOMCAT_HOME}/bin/shutdown.sh"
else
  doFail "Unknown tomcat command: ${1}"
fi